tests/login.js

```diff
@@ -77,4 +77,28 @@ test(`Login with invalid username and password`, async t => {
 
     await t.expect(receivedError).eql(expectedLoginError)
     await t.expect(isErrorVisible).eql(true)
-});
\ No newline at end of file
+});
+
+import { Selector, t, ClientFunction }  from "testcafe";
+import loginPage from '../Pages/loginPage'; 
+
+export const baseUrl = 'https://www.saucedemo.com/';
+const getURL = ClientFunction(() => window.location.href);
+
+fixture`Authentification`
+    .page(baseUrl)
+
+
+test(`Loading Login Page`, async t => { 
+    await t
+        .expect(getURL()).contains(baseUrl)
+        .expect(loginPage.loginButton.exists).ok();
+
+});
+
+
+test(`Login with valid user`, async t => {
+    await t
+        .expect(currentPage).eql('https://www.saucedemo.com/inventory.html')
```

Comment:
currentPage and isInventoryItemVisible are variables declared in a diferent test. You don't have access to the variables. If you run the tests, these should fail instantly. Please always try to execute the tests before commiting the code and see if you have any issues.